Misc/NEWS.d/next/Library/2023-10-17-16-11-03.gh-issue-52161.WBYyCJ.rst

```diff
@@ -0,0 +1,2 @@
+:meth:`cmd.Cmd.do_help` now cleans docstrings with :func:`inspect.cleandoc`

```

Comment:
I don't know if ":meth:`cmd.Cmd.do_help`" is correct, there definitely is no `cmd.Cmd.do_help` anchor in [the documentation](https://docs.python.org/3/library/cmd.html). Maybe I'd need to update the documentation to give `do_help` a separate section. I'm open to feedback here and in general.

Reply:
It seems to work. Let's not worry about it.